Dockerfile

```diff
@@ -36,6 +36,5 @@ RUN apk add --update --no-cache python3 && \
 	ln -sf python3 /usr/bin/python && \
 	python3 -m ensurepip
 USER preprocessing-moma
-RUN pip3 install --no-cache --upgrade pip lxml bagit==v1.8.1
```

Comment:
Isn't bagit necessary for the SIPCreation activity?

Reply:
Yeah, and you don't want to use that activity in here as it uses a custom Python script.